build/Linux+BSD/portable/RecipeArm

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# This file is designed to run in debian jessie docker images
+
+main() {
+
+  set -e # Halt on error
+  set -x # Be verbose
+
+  case $1 in
+    "--build-only")
+      shift #need to adjust parameter list so $makefile_overrides start at $1
+      build "$@"
+      ;;
+    "--package-only")
+      package "$@"
+      ;;
+    *)
+      build "$@"
+      package "$@"
+  esac
+
+  echo 'RecipeARM completed'
+}
+
+########################################################
+# BUILD MuseScore PORTABLE (Compile, not package)
+########################################################
+build() {
+  echo 'building'
+
+  cd MuseScore
+
+  make revision
+
+  LIBARM="/lib/arm-linux-gnueabihf"
+  CMAKE_TOOLCHAIN_FILE="/MuseScore/build/Linux+BSD/jessie-crosscompile-armhf.cmake" \
+  LD_LIBRARY_PATH="/usr$LIBARM:/usr$LIBARM/alsa-lib:/usr$LIBARM/pulseaudio:$LIBARM" \
+  make "$@" portable
```

Comment:
make using the cross compiler toolchain file, and with LD_LIBRARY_PATH pointed to all arm library directories.